<launch>
    <include file="$(find-pkg-share very_simple_robot_simulator)/launch/minimal_simulator_py.xml" >
        <arg name="initial_x" value="0.5" />
        <arg name="initial_y" value="0.5" />
        <arg name="initial_yaw" value="0.0" />
    </include>
    <node pkg="very_simple_robot_simulator" name="world_state_gui" exec="world_state_gui.py" output="screen" >
        <param name="map_file" value="ros2_ws/src/my_first_packege/my_first_packege/mapa.yaml" />
    </node>
    <include file="$(find-pkg-share very_simple_robot_simulator)/launch/lidar_simulator.xml" />
    <node pkg="my_first_packege" exec="robot_reactivo_test_C" name="robot_reactivo_test_C" >
        <param name="file_path" value="ros2_ws/src/my_first_packege/my_first_packege/mapa.pgm" />
    </node>
    <node pkg="my_first_packege" exec="localizacion_test_C" name="localizacion_test_C" >
            <param name="ruta" value="ros2_ws/src/my_first_packege/my_first_packege/" />
    </node>
    <node pkg="rviz2" exec="rviz2" name="rviz2" output="screen" args="-d ros2_ws/src/my_first_packege/rviz_localization.rviz" />
</launch>